app/controllers/application.js

```diff
@@ -78,6 +78,9 @@ export default Controller.extend({
    * Called when inspecting an object from outside of the ObjectInspector
   */
   activateMixinDetails(name, objectId, details, errors) {
+    if (this.mixinStack.objectAt(0) && this.mixinStack.objectAt(0).objectId === objectId) {
```

Comment:
What is this check preventing?

Reply:
If there are many renders, it will remove and add the same object in the inspector everytime, therefore making it impossible to go into properties and it also decreases the performance of the inspector. (If by some mistake you have 100ths of renders / s ...)

Reply:
If you end up adding another commit to this PR would you mind putting in a comment?

Reply:
Would it perhaps be a good idea to wrap in some sort of throttle or debounce? That way if we render hundreds of times, we could make it only actually render a few of those times.

Reply:
actually, I remember the real issue now. 
When a rerender happens, and i'm currently digging into an object, it will reset the object inspector. Throwing away my current inspection.
I will update the comment

Reply:
I'm not sure I have ever seen this behavior. Is there something in your app causing a lot of rerenders? Generally, once the page loads, I am able to use the object inspector without any issues.

Reply:
My concern here is something could be outdated if we just stop the updates.

Reply:
What is your app doing that is making it constantly refresh? I think that sounds like an issue with the app, not inspector.

Reply:
If we want to not refresh, we should check for deep equality of the properties in the object you are inspecting, if nothing changes, there is no need to refresh, but if it does we need to refresh or you are looking at the wrong info.

Reply:
it is an issue with the app, but i think we should be able to use ember-inspector during app development with app issues :)

but I just tested without this change, and its working fine now...

Reply:
I think we either should remove this check entirely or it should be a check that basically nothing on the instance changed. If all the properties remain the same, skip refreshing, etc. Might be easier to just remove for now.